policy/modules/admin/usbguard.te

```diff
@@ -0,0 +1,58 @@
+policy_module(usbguard, 1.0)
+
+########################################
+#
+# Declarations
+#
+
+## <desc>
+##	<p>
+##	Determine whether usbguard-daemon can modify its rules in
+##	/etc/usbguard.
+##	</p>
+## </desc>
+gen_tunable(usbguard_modify_rule_files, false)
+
+type usbguard_t;
+type usbguard_daemon_exec_t;
+init_daemon_domain(usbguard_t, usbguard_daemon_exec_t)
+
+type usbguard_etc_t;
+files_config_file(usbguard_etc_t)
+
+# /dev/shm
+type usbguard_tmpfs_t;
+files_tmpfs_file(usbguard_tmpfs_t)
+
+type usbguard_var_log_t;
+logging_log_file(usbguard_var_log_t)
+
+########################################
+#
+# Usbguard local policy
+#
+
+allow usbguard_t self:capability { chown dac_override dac_read_search fowner };
```

Comment:
the dac_override is due to a bug in the ipc library. A fix for this is upstream so with that you might be able omit the dac_override thereby greatly enhance security

Reply:
Seems to work without.